hadoop-ozone/ozone-recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/ContainerDBServiceProviderImpl.java

```diff
@@ -163,4 +165,21 @@ public Integer getCountForForContainerKeyPrefix(
     return prefixes;
   }
 
+  /**
+   * Iterate the DB to construct a unique set of containerIDs.
+   *
+   * @return List of containerIDs.
+   * @throws IOException
+   */
+  @Override
+  public Set<Long> getContainerIDList() throws IOException {
+    Set<Long> containerIDs = new HashSet<>();
```

Comment:
(Minor) LinkedHashSet may be better if we want to preserve containerId ordering on iteration.